### DISCUSSION TITLE
Looking for a working example or tutorial

### DISCUSSION THREAD

**Discussion by ingvardsen** [scripbox/snowplow-elixir-tracker#20] [General]
Great project. Though, are you aware of any working example or tutorial implementing it into a Phoenix project ?

**Comment by goodhamgupta** [comment]
Hi @ingvardsen,

We've successfully integrated this into a few projects at Scripbox. I'm not aware of any other open-source project using this tracker specifically, but there are a bunch of other trackers in languages(mentioned [here](https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/)), and there might be projects using these trackers. Also note that all trackers have the same API format, so you should be able to adapt this SDK to your usecase.

We would of course also deeply appreciate any contributions you would like to make towards this issue. Thanks!
